examples/pipe/dingo.ini

```
@@ -0,0 +1,129 @@
+################################################################################
+## Calibration arguments
+################################################################################
+
+# calibration-model=None
+# spline-calibration-envelope-dict=None
+# spline-calibration-nodes=10
+# spline-calibration-curves=1000
+
+
+################################################################################
+## Data generation arguments
+################################################################################
+
+# ignore-gwpy-data-quality-check=True
+# timeslide-file=None
+trigger-time= {{ production.meta['event time'] }} # GW150914 #
+# data-dict=None
+# data-format=None
+# allow-tape=None
+channel-dict = {H1:GWOSC, L1:GWOSC} #{ {% if production.meta['interferometers'] contains "H1" %}{{ production.meta['data']['channels']['H1'] }},{% endif %} {% if production.meta['interferometers'] contains "L1" %}{{ production.meta['data']['channels']['L1'] }},{% endif %}{% if production.meta['interferometers'] contains "V1" %}{{ production.meta['data']['channels']['V1'] }}{% endif %} }
```

Should we use the code that is commented here?

```suggestion
channel-dict = { {% if production.meta['interferometers'] contains "H1" %}{{ production.meta['data']['channels']['H1'] }},{% endif %} {% if production.meta['interferometers'] contains "L1" %}{{ production.meta['data']['channels']['L1'] }},{% endif %}{% if production.meta['interferometers'] contains "V1" %}{{ production.meta['data']['channels']['V1'] }}{% endif %} }
```